web/pandas/pdeps/accepted/0001-purpose-and-guidelines.md

```diff
@@ -0,0 +1,121 @@
+  is approved or not
+- Developers of pandas and other related projects, and experienced users. Their
+  feedback is highly encouraged and appreciated, to make sure all points of views
+  are taken into consideration
+- The wider pandas community, in particular users, who may or may not have feedback
+  on the proposal, but should know and be able to understand the future direction of
+  the project
+
+### PDEP authors
+
+Anyone can propose a PDEP, but in most cases developers of pandas itself and related
+projects are expected to author PDEPs. If you are unsure if you should be opening
+an issue or creating a PDEP, it's probably safe to start by
+[opening an issue](https://github.com/pandas-dev/pandas/issues/new/choose), which can
+be eventually moved to a PDEP.
+
+### Workflow
+
+#### Submitting a PDEP
+
+Proposing a PDEP is done by creating a PR adding a new file to `web/pdeps/accepted/`.
+The file is a markdown file, you can use `web/pdeps/accepted/0001.md` as a reference
+for the expected format.
+
+By default, we expect a PDEP will be accepted, so the PR of a PDEP should be done
+in the `accepted` but we will keep `Status: Under discussion` until it is ready to
+be merged. If a PDEP is finally rejected, its status and directory will be updated
+by the core team before merging, once the decision is made. Please make sure you
+select the option `Allow edits and access to secrets by maintainers` when opening the PR.
+
+#### Accepted PDEP
+
+A PDEP can only be accepted by the core development team, if the proposal is considered
+worth implementing. Decisions will be made based on the process detailed in the
+[pandas governance document](https://github.com/pandas-dev/pandas-governance/blob/master/governance.md).
+In general, more than one approval will be needed before the PR is merged. And
+there should not be any `Request changes` review at the time of merging.
+
+Once a PDEP is accepted, any contributions can be made toward the implementing the PDEP with an open-ended completion timeline . The
+pandas project development, with a mix of volunteers and developers paid from
+different sources, and development priorities are difficult to understand or
+forecast. For companies, institutions or individuals with interest in seeing a
+PDEP being implemented, or to in general see progress to the pandas roadmap,
+please check how you can help in the [contributing page](/contribute.html).
+
+#### Implemented PDEP
+
+Once a PDEP is implemented and available in the main branch of pandas, its
+status will be changed to implemented, so there is visibility that the PDEP
+is not part of the roadmap and future plans, but a change that it already
+happened. The first pandas version in which the PDEP implementation is
+available will also be included in the PDEP.
+
+#### Rejected PDEP
+
+A PDEP can be rejected when the final decision is that its implementation is
+not the best for the interests of the project. They are as useful as accepted
```

Comment:
This is to contrast rejected PDEPs against accepted PDEPs; i.e. "even rejected PDEPs are useful". I think the wording is okay, but maybe "Rejected PDEPs are just as useful as accepted PDEPs..." would make it more clear?

Reply:
Ah, I missed the first "as" in the sentence, so I read it as "they are useful as accepted PDEP", instead of "they are _as_ useful as accepted PDEPs", hence the confusion.